docs/logs/redaction/Program.cs

```diff
@@ -0,0 +1,38 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+
+namespace Redaction;
+
+public class Program
+{
+    public static void Main()
+    {
+        using var loggerFactory = LoggerFactory.Create(builder =>
+            builder.AddOpenTelemetry(options =>
+            {
+                options.AddProcessor(new MyRedactionProcessor());
```

Comment:
processors are run sync.. so it might affect latency if executing complex regex etc. But this might be okay for now. Unless we can show how to do this in Exporter (which wraps real exporter).